WordPress/wordpress-playground
# PHP Blueprints: Display progress

Renders the Blueprint execution progress information exposed in https://github.com/WordPress/blueprints/pull/56

## Follow-up work

* Pre-process the user-provided Blueprint.json and start pre-fetching data from the URL-based resources. Then, inform the user about the progress of those `fetch()` requests.

## Testing instructions

Go to http://localhost:5400/website-server/demos/php-blueprints.html
and confirm the progress information gets displayed like on this
screenshot:

<img width="550" src="https://github.com/WordPress/wordpress-playground/assets/205419/ca072946-ad67-483c-a1a3-7f2d146ed8eb">
